ex02.exs

```diff
@@ -81,24 +81,21 @@ end
 defmodule Ex02 do
 
   def new_counter(initial_value \\ 0) do
-    # « your code goes here»
+    { :ok, agent } = Agent.start_link( fn -> initial_value end) 
+     agent
   end
 
   def next_value(agent) do
-    # « your code goes here»
+    Agent.get_and_update(agent, fn total -> {total, total + 1} end)
   end
 
   @global_name :my_global_agent
 
   def new_global_counter(initial_value \\ 0) do
-    # « your code goes here»
-    # You can give an agent a name by adding the
-    # `name: @global_name` option to start_link
+    Agent.start_link( fn -> initial_value end, name: Globe)
   end
 
   def global_next_value do
-    # « your code goes here»
-    # and you can use that name instead of a pid
-    # in calls such as `get_and_update`
+    Agent.get_and_update(Globe, fn total -> {total, total+1} end) 
```

Comment:
My intent here was that you'd use @global_name as the name of the agent. That way you wouldn't have to duplicate the actual name.

# -1